packages/eas-cli/src/commands/submit.ts

```diff
@@ -89,6 +91,11 @@ export default class Submit extends EasCommand {
       default: false,
       description: 'Enable verbose logging for the submission process',
     }),
+    groups: Flags.string({
```

Comment:
Wouldn't it be better to have it in `eas.json`? Why/why not? 

It probably depends on how people use it 🤔